pkg/agent/openflow/pipeline.go

```diff
@@ -2584,16 +2683,20 @@ func pipelineClassifyFlow(cookieID uint64, protocol binding.Protocol, pipeline b
 		Done()
 }
 
-// igmpPktInFlows generates the flow to load CustomReasonIGMPRegMark to mark the IGMP packet in MulticastRoutingTable and sends
-// it to antrea-agent.
+// igmpPktInFlows generates the flow to load CustomReasonIGMPRegMark to mark the IGMP packet in MulticastIGMPEgressTable
```

Comment:
the coment is not accuate according to the code

Reply:
done